include/fmt/chrono.h

```diff
@@ -2190,51 +2186,51 @@ struct formatter<std::chrono::time_point<std::chrono::utc_clock, Duration>,
 
 template <typename Char> struct formatter<std::tm, Char> {
  private:
-  enum class spec {
-    unknown,
-    year_month_day,
-    hh_mm_ss,
-  };
-  spec spec_ = spec::unknown;
-  basic_string_view<Char> specs;
+  format_specs<Char> specs;
+  detail::arg_ref<Char> width_ref;
 
  protected:
-  template <typename It> FMT_CONSTEXPR auto do_parse(It begin, It end) -> It {
-    if (begin != end && *begin == ':') ++begin;
+  basic_string_view<Char> format_str;
+
+  FMT_CONSTEXPR auto do_parse(basic_format_parse_context<Char>& ctx)
+      -> decltype(ctx.begin()) {
+    auto begin = ctx.begin(), end = ctx.end();
+    if (begin == end || *begin == '}') return end;
+
+    begin = detail::parse_align(begin, end, specs);
```

Comment:
Just realized that `parse_align(begin, end, specs)` is marked as deprecated, should we use `parse_format_specs` instead?

Reply:
It's OK to use `parse_align` here. It might be easier to clean up and undeprecate `parse_align` instead of adjusting `parse_format_specs` but I need to think about it more.